### ISSUE TITLE
Don't get the trophies

### ISSUE THREAD

**Issue by MaGeBZz** [GTNewHorizons/GT-New-Horizons-Modpack#18025]
### Your GTNH Discord Username

magebzz_2

### Your Pack Version

2.7 Beta 3

### Your Server

Private Server

### Java Version

Java 21

### Type of Server

Don't know

### Your Expectation

You should get the trophie instantly if you obtain for example the "Coke Oven Brick" block

### The Reality

You have to pick it up two times in most cases until you get the trophy but it should be instantly, if you craft it, pick it up or whatever but after being in the inventory once at least

### Your Proposal

Fixing this? Lol

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying the bug still exists will prompt us to investigate and reopen it once we confirm your report.
- [X] I can reproduce this problem consistently by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.
- [X] I have asked other people and they confirm they also have this problem by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.

**Comment by PinkYuDeer** [comment]
2.7 Beta 4 have same question. The trophies I receive are usually picked up from the drops, rather than being taken from AE.

**Comment by MaGeBZz** [comment]
Yeah I had that even in beta 1 or so, at least before beta 3 I know for sure
Don't know why it is but it seems that you have to drop the unlock item for the trophy and get it again in your inventory to actually get the trophy
